sdk/ts/jest.config.mjs

```diff
@@ -0,0 +1,195 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+export default {
```

Comment:
remove all the comments

Reply:
https://github.com/fleek-network/ursa/pull/370/commits/5d5c77c0fae5a1a8d625360ff5bc91ee3c3a559d